nginx/templates/includes/cors.conf.template

```diff
@@ -0,0 +1,5 @@
+more_set_headers -s 403 'Content-Type' '${CORS_HEADER_403_CONTENT_TYPE}';
+more_set_headers -s 403 'Access-Control-Allow-Origin' '${CORS_HEADER_403_ALLOW_ORIGIN}';
+more_set_headers -s 403 'Access-Control-Max-Age' '${CORS_HEADER_403_MAX_AGE}';
+more_set_headers -s 403 'Access-Control-Allow-Methods' '${CORS_HEADER_403_ALLOW_METHODS}';
+more_set_headers 'Access-Control-Allow-Headers' '${CORS_HEADER_ACCESS_CONTROL_ALLOW_HEADERS}';
```

Comment:
I believe the default value should be "unset". According to the documentation, there's a list of headers that are allowed by default. In many cases, that list should not need to be extended. By supplying `*` as the value, we would allow _any_ header.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers